### PR TITLE
chore: use prerelease for browser

### PR DESCRIPTION
**Pull request by jrandolf-google**

### CODE REVIEW

**Review by Lightning00Blade — APPROVED** [review]

File: release-please-config.json [hunk 1]
```diff
@@ -9,11 +9,9 @@
       "component": "puppeteer-core"
     },
     "packages/testserver": {},
-    "packages/ng-schematics": {
-      "initial-version": "0.1.0"
-    },
+    "packages/ng-schematics": {},
     "packages/browsers": {
-      "initial-version": "0.0.1"
+      "release-as": "0.0.4"
```
Comment (OrKoN) [inline release-please-config.json:14]: this appears to be the wrong version. The next version is `0.0.5`, `0.0.4` is already published. Why was this change needed?
